client/landing/subscriptions/hooks/index.ts

```diff
@@ -1,5 +1,4 @@
 export { default as usePopoverToggle } from './use-popover-toggle';
-export { default as useSearch } from './use-search';
```

Comment:
`useSearch` was simplified, by removing the `debouncedSearchTerm` debouncing, reduced into a trivial `useState`, and inlined into the only module that used it.